sysutils/pfSense-pkg-nut/files/usr/local/pkg/nut.xml

```diff
@@ -0,0 +1,682 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE packagegui SYSTEM "../schema/packages.dtd">
+<?xml-stylesheet type="text/xsl" href="../xsl/package.xsl"?>
+<packagegui>
+	<copyright>
+	<![CDATA[
+/* $Id$ */
+/* ====================================================================================== */
+/*
+	nut.xml
+	part of pfSense (https://www.pfSense.org/)
+	Copyright (C) 2009-2015 ESF, LLC
+	Copyright (C) 2016 Sander Peterse <sander.peterse88@gmail.com>
+	All rights reserved.
+*/
+/* ====================================================================================== */
+/*
+	Redistribution and use in source and binary forms, with or without
+	modification, are permitted provided that the following conditions are met:
+
+
+	1. Redistributions of source code must retain the above copyright notice,
+	   this list of conditions and the following disclaimer.
+
+	2. Redistributions in binary form must reproduce the above copyright
+	   notice, this list of conditions and the following disclaimer in the
+	   documentation and/or other materials provided with the distribution.
+
+
+	THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+	INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+	AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+	AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+	OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+	SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+	INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+	CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+	ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+	POSSIBILITY OF SUCH DAMAGE.
+*/
+/* ====================================================================================== */
+	]]>
+	</copyright>
+	<name>nut</name>
+	<version>2.3.0</version>
+	<title>Services: NUT</title>
+	<savetext>Change</savetext>
+	<aftersaveredirect>/ups_status.php</aftersaveredirect>
+	<include_file>/usr/local/pkg/nut.inc</include_file>
+	<menu>
+		<name>NUT</name>
+		<tooltiptext>Set Network UPS Tools settings.</tooltiptext>
+		<section>Services</section>
+		<url>/ups_status.php</url>
+	</menu>
+	<service>
+		<name>nut</name>
+		<rcfile>nut.sh</rcfile>
+		<executable>upsmon</executable>
+		<description>UPS monitoring daemon</description>
+	</service>
+	<tabs>
+		<tab>
+			<text>UPS Status</text>
+			<url>/ups_status.php</url>
+		</tab>
+		<tab>
+			<text>NUT Settings</text>
+			<url>/pkg_edit.php?xml=nut.xml</url>
+			<active/>
+		</tab>
+	</tabs>
+	<advanced_options>enabled</advanced_options>
+	<fields>
+		<field>
+			<name>General Settings</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>UPS Monitoring</fielddescr>
+			<fieldname>monitor</fieldname>
+			<type>select</type>
+			<required/>
```

Comment:
Can you remove the 'required' ? Its not possible to disable nut with the "Disabled" option from line 86, which fails to save as it has no value.. (Or add another default empty option, with the choice to specifically disable nut?)